xml/Makefile.am

```diff
@@ -231,6 +231,22 @@ versions.rng: pacemaker-$(CIB_max).rng Makefile.am
 	$(AM_V_at)echo '  </start>' >> $@
 	$(AM_V_SCHEMA)echo '</grammar>' >> $@
 
+schemas:
+	@if [ -z "$$SCHEMAS" ] || [ -z "$$NEW_VERSION" ]; then \
```

Comment:
would be cool to default NEW_VERSION to most recent +1 to minor version

Reply:
Done.

If you set NEW_VERSION all the schemas will be copied to that version, which might not be ideal in all cases.

Do you want it to fail and tell the user to only use one schema at a time when using NEW_VERSION?

Reply:
Schemas are versioned as a collection rather than individually, so all copied schemas should get the same version, which should be the one after the current latest of all schemas (not just the copied ones)

Reply:
Updated to find the latest version of all the schemas.